src/patternfly/components/Popover/popover.scss

```diff
@@ -1,8 +1,14 @@
 .pf-c-popover {
   // Component
   --pf-c-popover--FontSize: var(--pf-global--FontSize--sm);
-  --pf-c-popover--MinWidth: calc(var(--pf-c-popover__content--PaddingLeft) + var(--pf-c-popover__content--PaddingRight) + #{pf-size-prem(300px)});
-  --pf-c-popover--MaxWidth: calc(var(--pf-c-popover__content--PaddingLeft) + var(--pf-c-popover__content--PaddingRight) + #{pf-size-prem(300px)});
+  --pf-c-popover--MinWidth:
+    calc(
```

Comment:
We generally try and keep the CSS functions on a single line when possible, unless it's very long and hard to read. Would you mind making this a single line and putting it inline with `--pf-c-popover--MinWidth`?

Reply:
This applies to the entire stylesheet, so I won't comment on other instances.

Reply:
This is fixed now.